lib/fauna/objects.rb

```diff
@@ -0,0 +1,111 @@
+module Fauna
+  ##
+  # A Ref.
+  #
+  # Reference: {FaunaDB Special Types}[https://faunadb.com/documentation#queries-values-special_types]
+  class Ref
+    # The raw ref string.
+    attr_accessor :value
+
+    ##
+    # Creates a Ref object from a string.
+    #
+    # +ref+:: A ref in string form.
+    def initialize(ref)
+      self.value = ref
+    end
+
+    ##
+    # Creates a new Ref object for the class of the current Ref.
+    def to_class
```

Comment:
I think previously i was using this and removed what it was being using for, so i can drop this.